truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -148,6 +144,11 @@ def RbConfig.expand(val, config = CONFIG)
     val.replace(newval) unless newval == val
     val
   end
+
+  CONFIG.each_value do |val|
+    RbConfig::expand(val)
+  end
```

Comment:
@eregon I think there are less chances for typos to use the MAKEFILE_CONFIG interpolation format to set the CONFIG values over string interpolation.

@chrisseaton I'm not sure if it could be lazily loaded because it already seems to be required when ruby starts.  I don't completely understand where lazy loading might happen though. For example, would lazy loading in post-boot help.

I believe that MRI tests are run with rubygems disabled. I think this was causing a `minitest/unit` failure in travis. I've added a commit to test disabling gems for MRI tests.

For reference, here is some more info regarding all the RbConfig values: http://idiosyncratic-ruby.com/42-ruby-config.html